examples/fancy/src/dynamic.js

```diff
@@ -0,0 +1,5 @@
+function dynamic() {
+  console.log('hi i am a dynamic import');
```

Comment:
@elifitch instead of console logging can we show a UI change in the `./fancy/src/app.js` component.

Reply:
Done 👍